apps/tenant-management-webapp/src/app/pages/admin/services/form/definitions/Rating.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { useState } from 'react';
+import { InputLabel } from '@mui/material';
+
+interface RatingProps {
+  id?: string;
+  value: number;
+  updateValue: (newValue: number) => void;
+}
+
+export const Rating: React.FC<RatingProps> = ({ id, value, updateValue }) => {
```

Comment:
This isn't for our production version, right?

Reply:
Oh, yeah. Anything with the word rating in it is can be removed